Quaver.Shared/Screens/Gameplay/Rulesets/Keys/HitObjects/GameplayHitObjectKeys.cs

```diff
@@ -477,6 +477,9 @@ private Texture2D GetHitObjectTexture(int lane, GameMode mode)
             {
                 var objects = Info.IsLongNote ? skin.NoteHoldHitObjects[lane] : skin.NoteHitObjects[lane];
 
+                if (Info.Color != -1)
```

Comment:
`Color` defaults to 0 for hitobjects, so anything that doesn't have their snap color overwritten is having their color overwritten to red

Reply:
forgot to commit the update to default Color to -1, locally this was changed